servicesContracts/requestEthereum_service.py

```diff
@@ -9,7 +9,16 @@
 
 class RequestEthereumService:
     def __init__(self):
-        pass
+        self.__web3Single = Web3Single.getInstance()
+        self._ipfs = Ipfs.getInstance()
+        self._abiRequestCore = requestCoreArtifact.abi
+        self._requestCoreServices = new RequestCoreService()
+        self._abiRequestEthereum = RequestEthereumArtifact.abi
+        if not requestEthereumArtifact.networks[self.__web3Single.networkName] :
+                    raiseException('RequestEthereum Artifact: no config for network : "' + self.__web3Single.networkName + '"')
```

Comment:
8 spaces?

Reply:
what 8 spaces? i didn't get

Reply:
The line `raiseException` (which should be `raise Exception`, didn't notice that before) has too many spaces.